lib/error-helper.spec.js

```diff
@@ -1,52 +1,49 @@
 'use strict'
 
-const chai = require('chai')
-const { assert, expect } = chai
+const { expect } = require('chai')
 const { checkErrorResponse } = require('./error-helper')
 const { NotFound, InvalidResponse } = require('../services/errors')
 
-chai.use(require('chai-as-promised'))
-
 describe('Standard Error Handler', function() {
   it('makes inaccessible badge', function() {
     const badgeData = { text: [] }
-    assert.equal(
-      true,
-      checkErrorResponse(badgeData, 'something other than null', {})
-    )
-    assert.equal('inaccessible', badgeData.text[1])
-    assert.equal('red', badgeData.colorscheme)
+    expect(checkErrorResponse(badgeData, 'something other than null', {})).to.be
+      .true
+    expect(badgeData.text[1]).to.equal('inaccessible')
+    expect(badgeData.colorscheme).to.equal('red')
   })
 
   it('makes not found badge', function() {
     const badgeData = { text: [] }
-    assert.equal(true, checkErrorResponse(badgeData, null, { statusCode: 404 }))
-    assert.equal('not found', badgeData.text[1])
-    assert.equal('lightgrey', badgeData.colorscheme)
+    expect(checkErrorResponse(badgeData, null, { statusCode: 404 })).to.be.true
+    expect(badgeData.text[1]).to.equal('not found')
+    expect(badgeData.colorscheme).to.equal('lightgrey')
   })
 
   it('makes not found badge with custom error', function() {
     const badgeData = { text: [] }
-    assert.equal(
-      true,
-      checkErrorResponse(badgeData, null, { statusCode: 404 }, 'custom message')
-    )
-    assert.equal('custom message', badgeData.text[1])
-    assert.equal('lightgrey', badgeData.colorscheme)
+    expect(
+      checkErrorResponse(
+        badgeData,
+        null,
+        { statusCode: 404 },
+        { 404: 'custom message' }
+      )
+    ).to.be.true
+    expect(badgeData.text[1]).to.equal('custom message')
+    expect(badgeData.colorscheme).to.equal('lightgrey')
   })
 
   it('makes invalid badge', function() {
     const badgeData = { text: [] }
-    assert.equal(true, checkErrorResponse(badgeData, null, { statusCode: 500 }))
-    assert.equal('invalid', badgeData.text[1])
-    assert.equal('lightgrey', badgeData.colorscheme)
+    expect(checkErrorResponse(badgeData, null, { statusCode: 500 })).to.be.true
+    expect(badgeData.text[1]).to.equal('invalid')
+    expect(badgeData.colorscheme).to.equal('lightgrey')
   })
 
   it('return false on 200 status', function() {
-    assert.equal(
-      false,
-      checkErrorResponse({ text: [] }, null, { statusCode: 200 })
-    )
+    expect(checkErrorResponse({ text: [] }, null, { statusCode: 200 })).to.be
+      .false
```

Comment:
Is this on a new line because of prettier/eslint?
I feel like it would look better on 1 line, but i'm fine with it either way.